components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -811,12 +811,17 @@ define([
                     this.getAgeLabel().show();
                     labels.push(this.getAgeLabel());
                 }
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
+                if (this.getCommentsLabel()) {
+                    this.getCommentsLabel().show();
+                    labels.push(this.getCommentsLabel());
+                }
             } else {
                 this.getNameLabel() && this.getNameLabel().hide();
                 this.getAgeLabel() && this.getAgeLabel().hide();
+                this.getExternalIDLabel() && labels.push(this.getExternalIDLabel());
```

Comment:
But if we need to show external ID?
This line originaly was implemented regardless anonimization, thats why it is both if and else blocks to preserve an original order of lines in pdf.